.github/workflows/test.yml

```diff
@@ -53,7 +59,7 @@ jobs:
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v3
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "${{ matrix.python-version }}"
```

Comment:
Let's avoid a bug ahead of time when someone has set python-version to the non-quoted string `3.10`, and have it YAML parsed as 3.1, and then transformed back to a string as `"3.1"`.